### ISSUE TITLE
Docs site is down

### ISSUE THREAD

**Issue by benr77** [dokku/dokku#5787]
### Description of problem

Looks like the documetation site is not working: http://dokku.viewdocs.io/dokku/

### Steps to reproduce

Visit http://dokku.viewdocs.io/dokku/

### dokku report $APP_NAME

Not applicable

**Comment by benr77** [comment]
Sorry am I looking at a legacy docs site on viewdocs.io? Search engines are still indexing all these pages. 

I can access https://dokku.com/docs/ just fine...

Cheers

**Comment by josegonzalez** [comment]
Do you have search examples for search engines looking at this? I can see what I can do to fix that.

**Comment by benr77** [comment]
3rd and 4th entry down
https://search.brave.com/search?q=dokku+crontab&source=desktop

I've just checked Google and can't see any viewdocs.io results, so it must just be the Brave search results.

Bing seems OK as well.

Sorry I didn't notice I was using the Brave search when I submitted the issue. Realise now that this isn't something that most users will be using so it's not so important.

When did the location of the docs site change? I can report to Brave if you like. I notice on the old URL we see the Heroku error message so it might be better if this was completely taken down so the old URLs return 404.
